documents/src/pages/elements/slider.md

```diff
@@ -141,6 +141,40 @@ ef-slider{
 <ef-slider min="0" max="100" from="60" to="80" step="20" range show-steps show-input-field></ef-slider>
 ```
 
+## Custom Marks
+Enhance sliders with custom marks using `ef-slider-marker` as children.
+
+::
+```javascript
+::slider::
+```
+```css
+ef-slider {
+  width:50%;
+  margin-left: 20px;
+}
+```
+```html
+<ef-slider step="25">
```

Comment:
We might want to add more examples such as no-label markers & non-text content e.g. ef-icon as a marker label.

Reply:
updated